src/oneD/StFlow.cpp

```diff
@@ -44,6 +45,11 @@ StFlow::StFlow(IdealGasPhase* ph, size_t nsp, size_t points) :
     // make a local copy of the species molecular weight vector
     m_wt = m_thermo->molecularWeights();
 
+	//make a local copy of species charge
+	for (size_t k = 0; k < m_nsp; k++ ){
```

Comment:
Please try to match formatting with surrounding code. For example: 
- 4 spaces per indentation level
- No tabs ever
- No space before close parentheses
- One space between `)` and `{`, e.g. in `for` and `if` statements.